### ISSUE TITLE
Add Command Line Argument or Option to Remove the Menu Music

### ISSUE THREAD

**Issue by PurpleCatMan** [Yellow-Dog-Man/Resonite-Issues#2091]
### Is your feature request related to a problem? Please describe.

It is annoying and frustrating that the Menu Music is bound to the multimedia slider and it pops up during syncing or quitting.

### Describe the solution you'd like

Please add an option to Mute the "Music" altogether with a slider or Command Line Argument.

### Describe alternatives you've considered

Bind the "Music" to the UI Slider.

**Comment by Electronus** [comment]
Menu music? Thee isn't any menu music. 

Do you mean the music in the local home?

**Comment by PurpleCatMan** [comment]
Yes, that one. I edited the title

**Comment by Electronus** [comment]
So I guess my question is why shouldn't it be on the multimedia slider. That's where you would expect any other world music yes?

**Comment by PurpleCatMan** [comment]
Obviously there are worlds and projects that have expected sound and audio to be played, but not outside of it. To me and the people I work with, the sound is distracting and frankly not needed.

**Comment by Electronus** [comment]
Well for now you could always just delete it, I'm not sure where it is the world hierarchy, but I suspect it won't be too hard to find, the local world is pretty simple.

**Comment by PurpleCatMan** [comment]
Thank you, but no.

If I install the Resonite Client every single time on several computers or several clients, I do not expect to go into every single inspector searching for the audio and to delete it manually and having to do so again if the computer or client has been reset.

Just as I am skipping the tutorial on cetrain computers and block the Cloud Home from being opened, I would like the Local Home to be silent, without affecting other projects that do use sounds.

This is why I am raising a ticket, as a User, wishing it to be simplified. 

I am not discussing this with you any further @Electronus, but thank you for your kind comments and eager help.

**Comment by Banane9** [comment]
> Well for now you could always just delete it, I'm not sure where it is the world hierarchy, but I suspect it won't be too hard to find, the local world is pretty simple.

That also doesn't work for the quitting world, since that's different from the local home.

**Comment by JackTheFoxOtter** [comment]
I assume cloud variables don't work when you're in your local home? Otherwise this could be a potential appraoch.

**Comment by Banane9** [comment]
> I assume cloud variables don't work when you're in your local home? Otherwise this could be a potential appraoch.

Those would have to be added to the default / template - but in the local home you also don't count as logged in, so I'd guess they wouldn't work.

**Comment by JackTheFoxOtter** [comment]
Well, there was limited support for local "cloud" variables added a while ago:

> [PRERELEASE] 2024.4.12.1259
Some more fixes and tweaks! One of the fixes is done by adding a new general feature - cloud variables now work (in limited manner) for users who are not logged in at all! They are stored in the local database. 
>
> This means any items (including the dash) that use them for configuration will now work too for those users.

So it could still function, you'd just have to disable it every time you reset your local database, but if it's just a button on your dash that should be fine.

**Comment by shiftyscales** [comment]
Would it be sufficient if there were an easy to press/visual toggle for the background music somewhere in the local world, @PurpleCatMan? That could be something the content team could add to the local home template.

**Comment by PurpleCatMan** [comment]
I dont think it would be sufficent, my most preferable option would be being able to disable it completely, before I start the client.

As I am giving a presentation on a new machine / client, even with the  -DoNotAutoLoadHome and -SkipIntroTutorials the noises just start blasting during startup as I am speaking, which is highly annoying.

I know its a separate case, but I wish there would be a "fast startup" and silent executable, which just loads into a black world and the main interface. Or even better, an argument like -LoadEmpty which would load and override the local home completely black, blank and silent, no visuals, so that during syncing there would also be no noises in the background.

Even if I setup Resonite launching into another world, during quitting and syncing there is always that noise in the local home.

**Comment by XDelta** [comment]
Not a solution to the audio part, but you can have the game load up a specific world on launch with `-Open Insert_URL_here - Open world at given URL on start (resrec)` https://wiki.resonite.com/Command_Line_Arguments, you can also change the cloud home world that normally is opened to be any world you want. You'll still briefly be in the local world as you always load into that.

Definitely agree with the sound being an amount annoying myself, I turned it off for my local home manually but it still exists for the exit screen and I can't as easily change that one.

**Comment by PurpleCatMan** [comment]
Thats why I would welcome a shorter alternative to a startup argument or a better solution, to having an option to remove the annoyance instead of going around it.

**Comment by PurpleCatMan** [comment]
Just a side idea, 
it would really be perfect having an Resonite executable next in the root folder ( just as an example something like ResoniteBlank.exe), which skips the Tutorial, initial setup, default in English, doesnt load the CloudHome, loads into a local home of silent black nothingness, just with the main interface, and also quits and syncs silently during quitting. Basically something that is stripped of all the visuals. I know y'all tailoring a user friendly experience, especially for newcomers, but it would be great something having something blank and empty, without the "experience" stuff, for the users who already know the FrooxEngine and around it.

Would you like a seperate ticket for that?
